WSUFoodiesApp/Student.java

```diff
@@ -1,12 +1,12 @@
 
 
-public class Student {
+public class User {
 
     String firstName;
     String lastName;
     String password;
     String email;
-    int idNum;
+
     
     class Bday {
         int day;

```

Comment:
There is no bday it's just int age and then getter and setter for the age. Fix constructors relating to this as well.